lib/rgeo/geos/ffi_factory.rb

```diff
@@ -328,8 +327,13 @@ def collection(elems)
       def multi_point(elems)
         elems = elems.to_a unless elems.is_a?(Array)
         elems = elems.map do |elem|
-          elem = RGeo::Feature.cast(elem, self, RGeo::Feature::Point,
-            :force_new, :keep_subtype)
+          elem = RGeo::Feature.cast(
+            elem,
+            self,
+            RGeo::Feature::Point,
+            :force_new,
+            :keep_subtype
+          )
           return unless elem
```

Comment:
@keithdoggett FYI, there's (at least) a place we return nil!

Reply:
Yeah we should probably fix it here since we're returning a geometry